### PR TITLE
SLES-11-SP2.xml: SLES guest's non-eth0 interface support 

### PR DESCRIPTION
**Pull request by aliang123**
shared.unattended.SLES-11-SP2.xml: Add script to create interface config file automatically(Bug 1361416)
shared.unattended.SLES-12.xml: Add script to create interface config file automatically(Bug 1361416)
virttest.remote.py: Correct scp output info(Bug 1361403)

### CODE REVIEW

**Comment by aliang123** [comment]
ID: 1361416, 1361403

**Comment by ngu-niny** [comment]
@aliang123 Suggest to split to two separate patches for the two bugs resolution.

File: virttest/remote.py [hunk 1]
```diff
@@ -797,12 +797,18 @@ def throughput_transfer(func):
     def transfer(*args, **kwargs):
         if "to" in func.__name__:
             msg = (
-                "Copy file from %s:%s to %s, " %
-                (args[0], args[6], args[5]))
-        else:
-            msg = (
+                #author:aliang,data:2016-8-15,bug:1361403
+                #"Copy file from %s:%s to %s, " %
+                #(args[0], args[6], args[5]))
```
Comment (ngu-niny) [inline virttest/remote.py]: No need to add the info at lines 800 ~ 802, the same applies at lines 807-809.

File: virttest/remote.py [hunk 2]
```diff
@@ -797,12 +797,18 @@ def throughput_transfer(func):
...
                 "Copy file from %s to %s:%s, " %
                 (args[5], args[0], args[6]))
+        else:
+            msg = (
+                #author:aliang,data:2016-8-15,bug:1361403
+                #"Copy file from %s to %s:%s, " %
+                #(args[5], args[0], args[6]))
+                "Copy file from %s:%s to %s, " %
+                (args[0], args[5], args[6]))
```
Comment (ngu-niny) [inline virttest/remote.py]: Should it be (args[0], args[6], args[5]))? :)

**Comment by aliang123** [comment]
Some error occured,  need to resend one new.

**Pull request closed without merging**